ocaml/testsuite/tests/typing-unique/overwriting.ml

```diff
@@ -487,9 +487,70 @@ Uncaught exception: File "ocaml/parsing/location.ml", line 1106, characters 2-8:
 (*******************************)
 (* Overwriting inlined records *)
 
-(* CR uniqueness *)
+type constructor_update = Con of { x : string; y : string }
+[%%expect{|
+type constructor_update = Con of { x : string; y : string; }
+|}]
+
+let update = function
+  | (Con _ as c) ->
```

Comment:
Yes. The current analysis is not able to determine the tag without a tag. I agree that it would be nice to figure out the tag from the type and will leave a CR.